src/ucp/stream/stream_send.c

```diff
@@ -225,11 +225,13 @@ static ucs_status_t ucp_stream_bcopy_multi(uct_pending_req_t *self)
                                                 sizeof(ucp_stream_am_hdr_t),
                                                 ucp_stream_pack_am_first_dt,
                                                 ucp_stream_pack_am_middle_dt,
-                                                ucp_stream_pack_am_last_dt);
+                                                ucp_stream_pack_am_last_dt, 1);
     if (status == UCS_OK) {
         ucp_request_t *req = ucs_container_of(self, ucp_request_t, send.uct);
         ucp_request_send_generic_dt_finish(req);
         ucp_request_complete_send(req, UCS_OK);
+    } else if (status == UCS_ERR_PENDING) {
+        status = UCS_OK; // remove from arbiter
```

Comment:
pls remove comment of make it with /* */

Reply:
comment is removed